datadog_checks_dev/datadog_checks/dev/tooling/commands/meta_commands/prometheus.py

```diff
@@ -0,0 +1,279 @@
+
+            echo_success('Info: ', nl=False, indent=indent)
+            echo_info(data.get('description', 'None'))
+
+            echo_info(options_text)
+
+            if choice_error:
+                echo_warning(choice_error)
+
+            echo_waiting(options_prompt, nl=False)
+
+            if num_options >= 9:
+                choice = ensure_unicode(input())
+            else:
+                # Terminals are odd and sometimes produce an erroneous null byte
+                choice = '\x00'
+                while choice == '\x00':
+                    choice = click.getchar().strip()
+                    try:
+                        choice = ensure_unicode(choice)
+                    except UnicodeDecodeError:
+                        choice = repr(choice)
+
+            if not choice:
+                choice = default_option
+
+            if choice == 't':
+                echo_info('Append .total')
+                for n in range(num_options):
+                    metric_options[n] += '.total'
+                options_text = get_options_text(metric_options)
+                continue
+            elif choice == 's':
+                echo_info('Skip')
+                echo_info('Skipped {}'.format(metric))
+                break
+            elif choice == 'q':
+                echo_info('Exit')
+                echo_warning('Exited at {}'.format(metric))
+                return
+
+            try:
+                choice = int(choice)
+            except Exception:
+                pass
+
+            if choice not in range(1, num_options + 1):
+                echo_info(u'{}'.format(choice))
+                choice_error = u'`{}` is not a valid option.'.format(choice)
+                continue
+            else:
+                choice_error = ''
+
+            option = metric_options[choice - 1]
+            echo_info(option)
+
+            data['dd_name'] = option
+
+            finished = True
+
+    metadata_file = path_join(output_dir, 'metadata.csv')
+    echo_waiting('\nWriting `{}`... '.format(metadata_file), nl=False)
+
+    metric_items = sorted(iteritems(metrics), key=lambda item: item[1]['dd_name'])
+    output_lines = [
+        'metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name\n'
+    ]
+    for metric, data in metric_items:
+        metric_name = data['dd_name']
+        metric_type = TYPE_MAP.get(data.get('type'), '')
+        metric_description = data.get('description', '')
+        if ',' in metric_description:
+            metric_description = '"{}"'.format(metric_description)
+
+        output_lines.append(
+            '{check}.{metric_name},{metric_type},,,,{metric_description},0,{check},\n'.format(
+                check=check,
+                metric_name=metric_name,
+                metric_type=metric_type,
+                metric_description=metric_description,
+            )
+        )
+
+    write_file_lines(metadata_file, output_lines)
```

Comment:
Are we both writing to file and copying to the clipboard here? Whats the use case for both? (Wondering if we should remove the copying to clipboard in this case (and the dependency it includes)

Reply:
We write csv, copy metric dict